components/playlist/browser/resources/components/playerControls.tsx

```diff
@@ -72,7 +72,7 @@ function Control ({
   return (
     <Button
       kind={kind}
-      size='mini'
```

Comment:
Just curious about why this didn't change to tiny or get removed (I think the default is medium which it probably would've been with the mini string, which isn't a known size)

Reply:
It was `jumbo` at first. Then I changed it to `mini` to reduce gap between buttons. Back then, it didn't change the icon's size. But this time, changing it to `tiny` made icons too small, while `jumbo` didn't change the gap between controls. So I reverted it to `jumbo` again.

Reply:
Sweet, just wanted to double check there wasn't something funky going on with the Buttons :D (or at least, any more funky than they already are).